src/models/IrcAction.ts

```diff
@@ -42,8 +42,9 @@ export class IrcAction {
                     break;
                 }
                 if (matrixAction.htmlText) {
-                    const text = ircFormatting.htmlToIrc(matrixAction.htmlText);
-                    const ircText = text ?? matrixAction.text; // fallback if needed.
+                    const ircText = ircFormatting.htmlToIrc(matrixAction.htmlText)
```

Comment:
Does `htmlToIrc` handle codeblocks?

Reply:
No, it will reject `<pre>` and `<code>`

Reply:
As in the contents of those elements? Sounds like we aught to include that. In fact, if you are sending code blocks over element, it will always be HTML?

Reply:
Oh I see `htmlToIrc` will return null if the HTML contained tags it doesn't recognise, so this is fine.